inc/Predictor.h

```diff
@@ -61,18 +70,18 @@ namespace PCOE {
         }
 
         /** @brief Set model pointer
-         *  @param value given model pointer
+         *  @param model given model pointer
          **/
-        virtual void setModel(PrognosticsModel* value) {
```

Comment:
I want to keep `value` here. Visual Studio doesn't like parameter names that shadow class members.